v2/pkg/catalog/loader/loader.go

```diff
@@ -181,8 +187,10 @@ func areTemplatesValid(store *Store, filteredTemplatePaths map[string]struct{})
 
 func areWorkflowOrTemplatesValid(store *Store, filteredTemplatePaths map[string]struct{}, isWorkflow bool, load func(templatePath string, tagFilter *filter.TagFilter) (bool, error)) bool {
 	areTemplatesValid := true
+
 	for templatePath := range filteredTemplatePaths {
-		if _, err := load(templatePath, store.tagFilter); err != nil {
+		_, err := load(templatePath, store.tagFilter)
```

Comment:
Why this change? IMO it's better to inline them whereas possible, because it limits the scope of the variable, making it less error prone and makes it less verbose. Go is verbose enough even without this...